app/models/mailer.rb

```diff
@@ -42,6 +42,18 @@ def create_message(m)
     end
   end
 
+  def debug_message(subj, txt)
+    params = {
+      from: quoted_address(HostUrl.outgoing_email_default_name, HostUrl.outgoing_email_address),
+      to: 'tech@bebraven.org',
```

Comment:
On Thu, Apr 07, 2016 at 10:26:20AM -0700, Brian Sadler wrote:
> Don't we have a config value somewhere for who we email?

Not in Canvas... but since it is to get an idea of the situation
for debugging anyway, we're the only logical recipients.